examples/server/utils.hpp

```diff
@@ -371,6 +371,7 @@ static json oaicompat_completion_params_parse(
     llama_params["repeat_last_n"]     = json_value(body,   "repeat_last_n",     default_sparams.penalty_last_n);
     llama_params["ignore_eos"]        = json_value(body,   "ignore_eos",        false);
     llama_params["tfs_z"]             = json_value(body,   "tfs_z",             default_sparams.tfs_z);
+    llama_params["n_keep"]            = json_value(body,   "n_keep",            0);
```

Comment:
Hello, thanks but @ggerganov  @ngxson  I worry this is actually [not OAI compatible](https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L5901) ?

Reply:
we can consider it as an "extension" to OAI, for example `tfs_z` or `mirostat` that we're having, they are not available on OAI.

Reply:
In fact this code is duplicated to the one inside `launch_slot_with_task`, I planned to refactor all of this to one place, maybe I'll this during weekend.